test/test_ops.py

```diff
@@ -1373,19 +1373,21 @@ def test_maxpool2d_bigger_stride(self):
           lambda x: torch.nn.functional.max_pool2d(x, kernel_size=(2,2), stride=stride),
           lambda x: Tensor.max_pool2d(x, kernel_size=(2,2), stride=stride))
 
-  @unittest.skipIf(Device.DEFAULT == "CUDA", "CUDA fails on this")
+  @unittest.skipIf(Device.DEFAULT in ["CUDA", "PYTHON"], "CUDA fails on this, and PYTHON is too slow")
```

Comment:
No need for this, these tensor sizes are stupidly big. Want to submit another PR making them smaller? Say:

(8, 2, 17, 14) or something

Reply:
#3417